generator/AWSPSGeneratorLib/HelpMaterials/Examples/ELB2/Register-ELB2Target.xml

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$targetDescription = New-Object Amazon.ElasticLoadBalancingV2.Model.TargetDescription
+$targetDescription.Port = 80
+$targetDescription.Id = 'i-0672a4c4cdeae3111
+
+Register-ELB2Target -TargetGroupArn 'arn:aws:elasticloadbalancing:us-east-1:123456789012:targetgroup/test-tg/a4e04b3688be1970' -Target $targetDescription -Region us-east-1'</code>
```

Comment:
You are missing a closing `'`

Can be simplified to `-Target @{Port = 80; Id = 'i-0672a4c4cdeae3111'}`